resources/zib/zib-VisualAcuity.xml

```diff
@@ -0,0 +1,301 @@
+      <alias value="VisusMeetHulpmiddel" />
+      <binding>
+        <strength value="extensible" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.12.19.4--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-visualacuity-v1.0-2020EN" />
+        <map value="NL-CM:12.19.6" />
+        <comment value="VisualAcuityMeasurementDevice" />
+      </mapping>
+    </element>
+    <element id="Observation.code">
+      <path value="Observation.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="16830007" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.effective[x]">
+      <path value="Observation.effective[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.effective[x]:effectiveDateTime">
+      <path value="Observation.effective[x]" />
+      <sliceName value="effectiveDateTime" />
+      <short value="VisualAcuityDateTime" />
+      <definition value="The date and time when the visual acuity was measured." />
+      <alias value="VisusDatumTijd" />
+      <type>
+        <code value="dateTime" />
+      </type>
+      <mapping>
+        <identity value="zib-visualacuity-v1.0-2020EN" />
+        <map value="NL-CM:12.19.4" />
+        <comment value="VisualAcuityDateTime" />
+      </mapping>
+    </element>
+    <element id="Observation.value[x]">
+      <path value="Observation.value[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.value[x]:valueQuantity">
+      <path value="Observation.value[x]" />
+      <sliceName value="valueQuantity" />
+      <short value="DecimalVisualAqcuity" />
+      <definition value="The measured decimal visual acuity." />
+      <alias value="DecimaleVisusWaarde" />
+      <type>
+        <code value="Quantity" />
+      </type>
+      <patternQuantity>
+        <system value="http://unitsofmeasure.org" />
+        <code value="{decimal}" />
+      </patternQuantity>
+      <minValueQuantity>
+        <value value="0" />
+          <system value="http://unitsofmeasure.org" />
+          <code value="{decimal}" />
+      </minValueQuantity>
+      <maxValueQuantity>
+        <value value="2" />
+          <system value="http://unitsofmeasure.org" />
+          <code value="{decimal}" />
+      </maxValueQuantity>
+      <mapping>
+        <identity value="zib-visualacuity-v1.0-2020EN" />
+        <map value="NL-CM:12.19.11" />
+        <comment value="DecimalVisualAqcuity" />

```

Comment:
'Aqcuity' should be replaced by 'Acuity' (this is an error in the zib itself).

Reply:
Idem https://bits.nictiz.nl/browse/ZIB-1445